Kickstarter-iOS/Views/Controllers/PledgeContinueViewController.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+import Library
+import Prelude
+
+final class PledgeContinueViewController: UIViewController {
+  // MARK: - Properties
+
+  private let continueButton = MultiLineButton(type: .custom)
```

Comment:
Are we still using `MultiLineButton` anywhere?

Reply:
Yeah we haven't officially done a pass to remove it. It's been removed here and there but not holistically.